### ISSUE TITLE
P_acman fails to build

### ISSUE THREAD

**Issue by BusinessKid** [Ponce/slackbuilds#52]

**Comment by BusinessKid** [comment]
Sorry. Omitted the Logfile
bash-4.4$ cat slackrepo_2019-03-05_16:58:37.log
Creating source repository:  /mnt/virtual/slackrepo/ponce/source
Cleaning /tmp/SBo ... done.
Checking whether git is clean ... no.
Indexing SlackBuilds ... done.
git repo: /mnt/virtual/slackrepo/ponce/slackbuilds-current
branch:   current
date:     Sat Mar  2 08:06:36 GMT 2019
revision: 8142e5f661606651e9782298933c833f346346ca (DIRTY)
title:    20190302.1 global branch merge.

================================================================================
games/pacman                                                            16:58:38
================================================================================

Calculating dependencies ... 
WARNING: games/pacman: Found installed package SDL2-2.0.9-x86_64-3 for SDL2 (not in repo)
WARNING: games/pacman: Found installed package SDL2_image-2.0.4-x86_64-1 for SDL2_image (not in repo)
WARNING: games/pacman: Found installed package SDL2_ttf-2.0.14-x86_64-2 for SDL2_ttf (not in repo)
WARNING: games/pacman: Found installed package SDL2_mixer-2.0.4-x86_64-1 for SDL2_mixer (not in repo)

Building games/pacman (add version 0.9) ------------------------------- 16:58:38
Downloading source files ...
/usr/libexec/slackrepo/functions.d/srcfunctions.sh: line 215:  /mnt/virtual/slackrepo/ponce/source/games/pacman/.version: No such file or directory
Verifying source files ... done.
Running pacman.SlackBuild ...
env MAKEFLAGS='2' nice -n 5 bash ./pacman.SlackBuild
games/pacman: pacman.SlackBuild failed (status 2)                       16:58:43
make: *** No rule to make target '2'.  Stop.
See /var/log/games/pacman/build.log
Unmounting chroot ... done.
:-( games/pacman FAILED )-:

================================================================================
SUMMARY                                                                 16:58:43
================================================================================

Logfile:     /var/log/slackrepo_2019-03-05_16:58:37.log
Runtime:     0h 0m 6s
Failed:      1
  games/pacman
Warnings:    4
  games/pacman: Found installed package SDL2-2.0.9-x86_64-3 for SDL2 (not in repo)
  games/pacman: Found installed package SDL2_image-2.0.4-x86_64-1 for SDL2_image (not in repo)
  games/pacman: Found installed package SDL2_mixer-2.0.4-x86_64-1 for SDL2_mixer (not in repo)
  games/pacman: Found installed package SDL2_ttf-2.0.14-x86_64-2 for SDL2_ttf (not in repo)

**Comment by Ponce** [comment]
there's no build log in this message, only slackrepo's log: try building it manually from the SlackBuild.
BTW, I tried building it here and it went fine.

**Comment by BusinessKid** [comment]
That didn't get very far. I had the chmod +x done on the slackbuild. Here's the log
bash-4.4# whoami
root
bash-4.4# pacman/pacman.SlackBuild
bash: pacman/pacman.SlackBuild: Permission denied
bash-4.4# ls -l pacman
total 20
-rw-r--r-- 1 root root  205 Mar  4 17:22 README
-rw-r--r-- 1 root root  130 Mar  4 17:22 doinst.sh
-rwxr-xr-x 1 root root 3311 Mar  4 17:22 pacman.SlackBuild
-rw-r--r-- 1 root root  339 Mar  4 17:22 pacman.info
-rw-r--r-- 1 root root  648 Mar  4 17:22 slack-desc
bash-4.4# 
I checked the directory. I have installed stuff (MasterPDF, google-earth, built drawtiming)
